roles/core/nfs_client/defaults/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+nfs_client_role_version: 1.0.4
+
```

Comment:
This is a variable internal to the role, already in `vars/main.yml`. Can you remove these 2 lines?

Reply:
OK, GitHub displayed these lines in the review while you removed those in a later commit... My bad.